src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -57,7 +57,8 @@ export const App = ({ displayToggle, toggleLoginModal }) => {
 
       apiRequest('/form1095_bs/available_forms')
         .then(response => {
-          if (response.errors || response.availableForms.length === 0) {
```

Comment:
Removing the response.errors check because it shouldn't be possible for non-success cases and there are no cases where errors imply that available forms aren't found. System error would be more accurate.

Reply:
This fix works great!